js/app/init.js

```diff
@@ -447,6 +447,17 @@ define([], () => {
                             location: 0.35
                         }]
                 ]
+            },  
+            wh_rolling: {
+                cssClass: 'pf-map-connection-preserve-mass',
+                overlays: [
+                    ['Label',
+                        {
+                            label: '<i class="fas fa-fw fa-sync-alt"></i>&nbsp;rolling',
+                            cssClass: ['pf-map-component-overlay', 'rolling'].join(' '),
+                            location: 0.35
```

Comment:
I suspect this value needs changing so it doesn't overlap with the save mass entry?

Reply:
Have you tried setting both the Rolling and Save Mass flags?

Reply:
![image](https://github.com/user-attachments/assets/7805b0d7-a81b-486a-bec0-5442372e220f)

This is with the size indicator set to 0.5.